### ISSUE TITLE
[Bug] PlayNext list not loading

### ISSUE THREAD

**Issue by alessandrobaldazzi** [SparrowBrain/Playnite.PlayNext#31]
This is the part of the log that I think is most relevant to the problem:

```
13-10 17:54:32.057|INFO |PlayNext#GameActivityExtension:1 games with recent activity found
13-10 17:54:32.375|DEBUG|PlayNext#GameActivityExtension:Games with playtime: 9
13-10 17:54:32.460|ERROR|PlayNext#PlayNext:Failure while refreshing data.
System.InvalidOperationException: Failed to compare two elements in the array. ---> System.NotSupportedException: Not supported series calculation method
   at PlayNext.Model.Score.GameScore.GameScoreBySeriesCalculator.<>c__DisplayClass5_0.<GetScoreMultiplierByPositionInSeries>b__0(Game a, Game b) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\GameScoreBySeriesCalculator.cs:line 108
   at System.Array.FunctorComparer`1.Compare(T x, T y)
   at System.Collections.Generic.ArraySortHelper`1.SwapIfGreater(T[] keys, IComparer`1 comparer, Int32 a, Int32 b)
   at System.Collections.Generic.ArraySortHelper`1.IntroSort(T[] keys, Int32 lo, Int32 hi, Int32 depthLimit, IComparer`1 comparer)
   at System.Collections.Generic.ArraySortHelper`1.IntrospectiveSort(T[] keys, Int32 left, Int32 length, IComparer`1 comparer)
   at System.Collections.Generic.ArraySortHelper`1.Sort(T[] keys, Int32 index, Int32 length, IComparer`1 comparer)
   --- End of inner exception stack trace ---
   at System.Collections.Generic.ArraySortHelper`1.Sort(T[] keys, Int32 index, Int32 length, IComparer`1 comparer)
   at System.Array.Sort[T](T[] array, Int32 index, Int32 length, IComparer`1 comparer)
   at System.Collections.Generic.List`1.Sort(Comparison`1 comparison)
   at PlayNext.Model.Score.GameScore.GameScoreBySeriesCalculator.GetScoreMultiplierByPositionInSeries(Guid gameId, OrderSeriesBy orderSeriesBy, Dictionary`2 seriesWithGames, KeyValuePair`2 seriesScore) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\GameScoreBySeriesCalculator.cs:line 83
   at PlayNext.Model.Score.GameScore.GameScoreBySeriesCalculator.<>c__DisplayClass3_0.<GetGamesWithSummedUpScores>b__1(KeyValuePair`2 x) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\GameScoreBySeriesCalculator.cs:line 61
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at PlayNext.Model.Score.GameScore.GameScoreBySeriesCalculator.GetGamesWithSummedUpScores(OrderSeriesBy orderSeriesBy, Dictionary`2 gamesWithSeriesScores, Dictionary`2 seriesWithGames) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\GameScoreBySeriesCalculator.cs:line 58
   at PlayNext.Model.Score.GameScore.GameScoreBySeriesCalculator.Calculate(OrderSeriesBy orderSeriesBy, IEnumerable`1 games, Dictionary`2 attributeScore) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\GameScoreBySeriesCalculator.cs:line 23
   at PlayNext.Model.Score.GameScore.FinalGameScoreCalculator.<>c__DisplayClass10_0.<Calculate>b__5() in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\FinalGameScoreCalculator.cs:line 58
   at PlayNext.Model.Score.GameScore.FinalGameScoreCalculator.CalculateWeightedScores(Func`1 scoreCalculationFunction, Single weight) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\FinalGameScoreCalculator.cs:line 107
   at PlayNext.Model.Score.GameScore.FinalGameScoreCalculator.Calculate(IEnumerable`1 games, Dictionary`2 attributeScore, GameScoreWeights gameScoreCalculationWeights, OrderSeriesBy orderSeriesBy, Int32 desiredReleaseYear, TimeSpan desiredGameLength) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\GameScore\FinalGameScoreCalculator.cs:line 58
   at PlayNext.Model.Score.TotalScoreCalculator.Calculate(PlayNextSettings savedSettings) in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\Model\Score\TotalScoreCalculator.cs:line 76
   at PlayNext.PlayNext.<<RefreshPlayNextData>b__39_0>d.MoveNext() in C:\Users\Qwx\Documents\src\Playnite.PlayNext\PlayNext\PlayNext.cs:line 215
```

Other aspects I've noted and tried:

1) I can access the screen where usually it should show me the suggested next games, but it's just completely empty. I can also access the configuration and modify it.
2) It doesn't seem to be dependent on the theme, I tried a bunch of them and doesn't work even with the default playnite one
3) I also tried disabling all the other extensions, but it also didn't sort any effect
4) I tried making a clean reinstall of the extension deleting the data folder and uninstalling it
5) Between the time it broke and now it updated once, but with no changes on the problem
6) I couldn't pinpoint the exact moment it stopped working, but I don't remember adding anything in particular, maybe just updating other extensions

Here's the complete log if needed:
[extensions.log](https://github.com/user-attachments/files/17356343/extensions.log)

**Comment by SparrowBrain** [comment]
It looks like you settings could be malformed. Could you share your config.json for PlayNext? You can find it by navigating to installed extensions, selecting PlayNext and clicking on Data folder.

![image](https://github.com/user-attachments/assets/c39fc025-4f6f-4919-b795-2c4d7a03822c)

I might be slow to respond / fix it, since I have little time. But I suspect, that if you delete your config.json it should start working.

**Comment by SparrowBrain** [comment]
I am dumbfounded, that I managed to leave such a bug.

Basically you have Series ordered by Release date in the settings (the default) and you have at least two games in a series that don't have any release dates.

Should be fixed in v1.5.3

**Comment by alessandrobaldazzi** [comment]
Just downloaded the latest version, it's working perfectly.
Thanks man 🙌🏼
